unleash-yggdrasil/src/strategy_grammar.pest

```diff
@@ -10,7 +10,7 @@ semver = @{ ASCII_DIGIT+ ~ "."  ~ ASCII_DIGIT+ ~ "."  ~ ASCII_DIGIT+ ~ semver_pa
     semver_build    = @{ "+" ~ semver_fragment }
         semver_fragment = @{ (ASCII_ALPHANUMERIC | "-")+ ~ ("." ~ (ASCII_ALPHANUMERIC | "-")+)* }
 
-date = @{ quad_fragment ~ "-" ~ pair_fragment ~ "-" ~ pair_fragment ~ "T" ~ pair_fragment ~ ":" ~ pair_fragment ~ ":" ~ pair_fragment ~ "." ~ trip_fragment ~ timezone_fragment }
+date = @{ quad_fragment ~ "-" ~ pair_fragment ~ "-" ~ pair_fragment ~ "T" ~ pair_fragment ~ ":" ~ pair_fragment ~ ":" ~ pair_fragment ~ ("." ~ trip_fragment)? ~ timezone_fragment }
```

Comment:
Dotnet does a thing where it truncates milliseconds when parsing dates. This isn't an existing bug in Yggdrasil since that's controlled by Yggdrasil's own date handling, but when you're using Dotnet, this doesn't parse. This allows us to use dates without millisecond precision

Reply:
Haven't we all? xD

Never change, .NET, never change